test/test.js

```diff
@@ -97,4 +97,37 @@ describe('solidity-repl', () => {
         assert.equal(result, 10)
       })
   })
+
+  it('should execute special command msg', () => {
+    return repl('msg')
+      .then(result => {
+        assert.equal(Object.keys(JSON.parse(result)).length, 5)
```

Comment:
I recommend adding `const parsedResult = JSON.parse(result)` or similar to avoid parsing the result more than once.